packages/renderer/src/TelemetryService.ts

```diff
@@ -46,4 +51,8 @@ export class TelemetryService {
       }
     }, 200);
   }
+
+  public handleExtensionEvent(event: ExtensionTelemetryEventType, id: string, featured: boolean, error: string) {
+    window.telemetryTrack(event, { id, featured, error });
+  }
```

Comment:
IMHO helper for extension shouldn't be in the service, helper is where the type is known

Reply:
Telemetry is handled backend in a general manner